docs/loadflow/loadflow.md

```diff
@@ -92,6 +96,52 @@ $$v_{b_1} + s \cdot q_{svc} = V^{c}_{b_1}$$
 
 Where $s$ is the slope of the static var compensator.
 
+### Computing HVDC power flow
+
+#### LCC converters
+
+LCC converters can be considered as fixed loads in the load flow. Indeed, on one side of the line is the rectifier station, and on the other side of the line is the inverter station. 
+The active power flows from the rectifier station to the inverter station, is fixed, and equals to a target value $P$ (AC side). The active power flow at each station at AC side is given by:
+  - $P_{rectifier}= P$
+  - $P_{inverter}= (1 - LossFactor_{inverter}) * ((1 - LossFactor_{rectifier}) * (P - P_{LineLoss}))$
+
+Power flows are in load sign convention, the active power at the rectifier AC terminal is positive and the active power at the inverter AC terminal is negative.
+The HVDC line losses $P_{LineLoss}$ are described in a dedicated section further below.
+
+The reactive power consumption of each converter on AC side is determined by the configured converter power factor of the converter station in the grid model, representing the ratio between active power $P$ and apparent power $S$.
+For each converter, its target reactive power is given by:
+  - $Q=\mid P*\tan(\cos(powerfactor))\mid$
+
+Note that LCC converters are always absorbing reactive power.
+
+#### VSC converters
+
+VSC converters are self-commutated converters that can be assimilated to generators in the loadflow. There can be two main modes to control active power flow through the line:
+- In **active power setpoint** mode, as for LCC converters, on one side of the line is the rectifier station, and on the other side of the line is the inverter station. The active power flow
+from the rectifier station to the inverter station is fixed and equals to a target value $P$ (AC side). The active power flow at each station at AC side is given by:
+  - $P_{rectifier}= P$
+  - $P_{inverter}= (1 - LossFactor_{inverter}) * ((1 - LossFactor_{rectifier}) * (P - P_{LineLoss}))$
+
+- In **AC emulation** mode, the active power flow between both stations is given by: $P = P_0 + k~(\theta_1 - \theta_2)$ 
+with $\theta_1$ and $\theta_2$ being the voltage angles at the bus connection for each converter station, and $P_0$ and $k$ being fixed parameters for the HVDC line. 
+If $P$ is positive, the converter station 1 is controller, else it is converter station 2. For example, if station 1 is controller, the
+active power flow at each station is given by the formula below (HVDC line losses are described in the next paragraph):
+  - $P_{controller} = P_0 + k~(\theta_1 - \theta_2)$
+  - $P_{noncontroller} = (1 - LossFactor_{noncontroller}) * ((1 - LossFactor_{controller}) * (P_0 + k~(\theta_1 - \theta_2) - P_{LineLoss}))$
+
+The HVDC line losses are described in a dedicated section further below.
+
+In both control modes (active power setpoint mode or in AC emulation mode), the target value $P$ is bounded by a maximum active power $P_{max}$ that can possibly be different from one direction to another.
```

Comment:
> maximum active power $P_{max}$ that can possibly be different from one direction to another.

really ? Isn't it just `HvdcLine.maxP` ?

Reply:
ok found out... there is an extension available, I am adding the doc for this

Reply:
Yes indeed, it is better to add doc for the extension HvdcOperatorActivePowerRange. Thanks !